src/rust/analyzer-dispatcher/tests/analyzer_dispatcher_integration_tests.rs

```diff
@@ -0,0 +1,115 @@
+#![cfg(feature = "integration_tests")]
+
+use std::time::Duration;
+
+use bytes::Bytes;
+use e2e_tests::{
+    test_fixtures,
+    test_utils::context::E2eTestContext,
+};
+use plugin_work_queue::test_utils::scan_analyzer_messages;
+use rust_proto::graplinc::grapl::api::pipeline_ingress::v1beta1::PublishRawLogRequest;
+use test_context::test_context;
+
+#[test_context(E2eTestContext)]
+#[tokio::test]
+async fn test_analyzer_dispatcher_inserts_job_into_plugin_work_queue(
+    ctx: &mut E2eTestContext,
+) -> eyre::Result<()> {
+    let tenant_id = ctx.create_tenant().await?;
+
+    let event_source_id = ctx
+        .create_event_source(
+            tenant_id,
+            "analyzer-dispatcher-test-event-source".to_string(),
+            "analyzer dispatcher test event source".to_string(),
+        )
+        .await?;
+
+    let generator_id = ctx
+        .create_generator(
+            tenant_id,
+            "analyzer dispatcher test sysmon generator".to_string(),
+            event_source_id,
+            test_fixtures::get_sysmon_generator()?,
+        )
+        .await?;
+
+    ctx.deploy_generator(generator_id).await?;
+
+    let analyzer_id = ctx
+        .create_analyzer(
+            tenant_id,
+            "analyzer dispatcher test analyzer".to_string(),
+            Bytes::from("fake"), // FIXME: use a real analyzer
```

Comment:
For generator/analyzers, since we're not deploying them I do think it's totally fine to just have Bytes::from("fake") instead of the real deal - but whatevs